src/main/java/org/mamute/list/TabsHelper.java

```diff
@@ -0,0 +1,47 @@
+package org.mamute.list;
+
+import br.com.caelum.vraptor.http.route.Router;
+import net.vidageek.mirror.dsl.Mirror;
+import org.apache.commons.lang.WordUtils;
+import org.mamute.controllers.ListController;
+
+import javax.inject.Inject;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class TabsHelper {
+
+	private final Router router;
+
+	@Inject
+	public TabsHelper(Router router) {
+		this.router = router;
+	}
+
+	public Tab tabForType(Tab.Type tabType) {
+		String localizationKey = "menu.top." + tabType.getStringValue();
+		String methodName = "top" + WordUtils.capitalize(tabType.getStringValue());
+		return new Tab(tabType, localizationKey, urlForMethod(methodName));
```

Comment:
Again, I'm just trying to eliminate this helper class. Of course you could create a factory or so for that if you don't want to put that in the model(which is fair enough)